pkgs/applications/video/go2tv/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildGoModule, fetchFromGitHub, xorg, libglvnd, pkg-config }:
+
+buildGoModule rec {
+  pname = "go2tv";
+  version = "1.13.0";
+
+  src = fetchFromGitHub {
+    owner = "alexballas";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-ZHKfBKOX3/kVR6Nc+jSmLgfmpihc6QMb6NvTFlsBr5E=";
+  };
+
+  vendorSha256 = "sha256-msXfXFWXyZeT6zrRPZkBV7PEyPqYkx+JlpTWUwgFavI=";
+
+  doCheck = false;
+
+  buildInputs = [
+    xorg.libX11.dev
+    xorg.libXcursor.dev
+    xorg.libXrandr.dev
+    xorg.libXinerama.dev
+    xorg.libXi.dev
+    xorg.libXext.dev
```

Comment:
dev output should be choosen automatically

Reply:
removed `.dev`